services/mapit/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.7'
+  
+volumes:
+  mapit-pg:
+
+x-mapit: &mapit
+  build:
+    context: .
+    dockerfile: services/mapit/Dockerfile
+  image: mapit
+  working_dir: /mapit
+
+services:
+  mapit-pg:
+    image: mdillon/postgis:9.6-alpine
+    volumes:
+      - mapit-pg:/var/lib/postgresql/data
+
+  mapit:
+    <<: *mapit
+    volumes:
+        - ~/govuk/mapit:/mapit
+    depends_on:
+      - mapit-pg
+      - nginx-proxy-app
+    working_dir: /mapit
+    environment:
+      MAPIT_DB_PASS: ''
```

Comment:
I tried to remove it but it breaks the DB. I think we need it because of https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk/manifests/apps/mapit.pp#L78

Reply:
I wonder if we should be [setting `GOVUK_ENV` to `development`](https://github.com/alphagov/mapit/blob/462da7a05c1b0011ea0e0a5354c7e29794a73a12/project/settings.py#L91) and then doing something similar to what we did for `MAPIT_DB_HOST` to `MAPIT_DB_PASS` (with allowing it to come from an environment variable)? 

The advantage of setting the environment to development is that we get `DEBUG = True` and a hard coded secret key (so we don't need to set it an environment variable below).